teams/README.md

```diff
@@ -0,0 +1,12 @@
+# Teams and projects
+
+The Tilt team is split into multiple (currently 2 teams) that each work on an individual project, typically for a 5 or 6 weeks sprint, in a synced cadence. At the end of the sprint, teams are disbanded and new teams and projects are formed for the following sprint.
+
+A project has a single charter outlining the business background and motivation for the upcoming sprint. It also contains a set of preliminary goals, which serve as success criteria for the project. Once a team is assigned to a project, they are responsible for further definining the goals, aiming to make them more objective and measurable. They are also responsible for scoping out work for the duration of the sprint. Teams are expected to move quickly and be responsive to changing user needs. As such, the charter (including it's goals and work scope) may change during the course of the sprint.
```

Comment:
```suggestion
A project has a single charter outlining the business background and motivation for the upcoming sprint. It also contains a set of preliminary goals, which serve as success criteria for the project. Once a team is assigned to a project, they are responsible for refining the charter and further defining the goals, aiming to make them more objective and measurable. They are also responsible for scoping out work for the duration of the sprint. Teams are expected to move quickly and be responsive to changing user needs. As such, the charter (including its goals and work scope) may change during the course of the sprint.
```

Reply:
(at least, this was how we did it last round--the charter was just a proposal and the team put our heads together and tweaked it until it accurately represented the work we wanted to do. I assume that's still the case?)

Reply:
This is great. Thanks @maiamcc ! And thanks for using the code suggestion feature!